### PR TITLE
brave: Update to version 0.60.48

### PR DESCRIPTION
**Pull request by rzpio**
Hey, it's my first PR here. I think scoop is very cool, thanks for that! :heart: 

Updated the Brave Browser to version 0.60.48. Refers to #1728. Some changes had to be made because Brave is now based on Chromium and moved to another Git repository.

### CODE REVIEW

File: bucket/brave.json [hunk 1]
```diff
@@ -1,45 +1,45 @@
...
     },
-    "extract_dir": "lib\\net45",
-    "extract_to": "brave",
-    "bin": "brave\\Brave.exe",
-    "post_install": "Copy-Item -force \"$dir\\brave\\squirrel.exe\" \"$dir\\update.exe\"",
+    "pre_install": [
+        "extract_7zip \"$dir\\chrome.7z\" \"$dir\"",
+        "Move-Item $dir\\Chrome-bin\\* $dir",
+        "Remove-Item $dir\\Chrome-bin",
+        "Remove-Item $dir\\chrome.7z"
```
Comment (Ash258) [inline bucket/brave.json]: Escape path with `$dir` in all lines above.

File: bucket/brave.json [hunk 2]
```diff
@@ -1,45 +1,45 @@
...
-            "brave\\Brave.exe",
+            "brave.exe",
             "Brave"
         ]
     ],
     "checkver": {
-        "url": "https://laptop-updates.brave.com/1/releases/dev/$majorVersion.$minorVersion.0/winx64?accept_preview=false",
-        "jp": "$.version"
+        "url": "https://raw.githubusercontent.com/brave/brave-browser/master/CHANGELOG.md",
+        "re": "(\\[(?<version>\\d+\\.\\d+\\.\\d+)\\])"
```
Comment (Ash258) [inline bucket/brave.json]:
```suggestion
        "regex": "\\[([\\d\\.]+)\\]"
```

**Comment by rzpio** [comment]
Thanks for your review, @Ash258. I implemented the changes.

File: bucket/brave.json [hunk 3]
```diff
@@ -1,45 +1,46 @@
...
         }
     },
-    "extract_dir": "lib\\net45",
-    "extract_to": "brave",
-    "bin": "brave\\Brave.exe",
-    "post_install": "Copy-Item -force \"$dir\\brave\\squirrel.exe\" \"$dir\\update.exe\"",
+    "pre_install": [
+        "extract_7zip \"$dir\\chrome.7z\" \"$dir\"",
+        "Move-Item \"$dir\\Chrome-bin\\*\" \"$dir\"",
+        "Remove-Item \"$dir\\Chrome-bin\"",
```
Comment (Ash258) [inline bucket/brave.json]: Chrome-bin is folder. Recurse parameter is needed. Remove-item accept array. Next line could be deleted.

```suggestion
        "Remove-Item \"$dir\\Chrome-bin\", \"$dir\\chrome.7z\" -Recurse"
```

**Comment by rzpio** [comment]
@Ash258, haha. 😄 Thank you, I implemented the changes.

**Pull request merged**